### DISCUSSION TITLE
Permission denied error while using Docker

### DISCUSSION THREAD

**Discussion by DendeberiaOleksandr** [givanz/Vvveb#77] [General]
Hey, I have following errors while using docker-compose:
Any ideas ?

![image](https://github.com/givanz/Vvveb/assets/57542777/8ec06158-37c3-42c4-8654-75c67d7cafa9)

**Comment by givanz** [comment]
Hi

Thanks for the bug report, I'm unable to reproduce the bug on Linux Mint but I think this happens because php can't create `/storage/model/admin/plugins` folder or that it created them with less permissions, probably some OS specific permission issue.

I added the empty folders in the last commit https://github.com/givanz/Vvveb/commit/e95807c56a899152ae5581d1bdf451be47653d48 to avoid this.

You can pull the change or manually create the `plugins/contactform` folders inside `/storage/model/admin/` or if they already exists set higher permissions, you can temporarily set 0777 to see if it works.

Let me know if it still doesn't work.
